### PR TITLE
fixed variant loading and variant parquet filtering

### PR DESCRIPTION
**Pull request by erictu**

### CODE REVIEW

**Comment by erictu** [comment]
Jenkins, test this please.

**Comment by erictu** [comment]
Jenkins, test this please

File: mango-cli/src/main/webapp/js/overall.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ if (featuresExist === true) {
 }
 
 // send pixel size for bining and initialize autocomplete
-var initJson =  "/init/" + $(".samples").width();
+var initJson =  "/init/" + Math.round($(".samples").width());
```
Comment (akmorrow13) [inline mango-cli/src/main/webapp/js/overall.js]: why are you rounding?

Comment (erictu) [inline mango-cli/src/main/webapp/js/overall.js]: Backend throws a fit if you don't feed it a round number.

**Pull request merged**